Documentation/security/policy/language.rst

```diff
@@ -503,6 +504,23 @@ but not CIDR prefix ``10.96.0.0/12``
 
         .. literalinclude:: ../../../examples/policies/l3/cidr/cidr.json
 
+.. _cidr_select_nodes:
+
+Selecting nodes with CIDR / ipBlock
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../../beta.rst
+
+By default, CIDR-based selectors do not match in-cluster entities (pods or nodes).
+Optionally, the policy engine can be directed to select nodes by CIDR / ipBlock.
+This requires Cilium to be configured with ``--policy-cidr-match-mode=nodes`` or
```

Comment:
Use active voice, address your reader as "you"
```suggestion
Optionally, you can direct the policy engine to select nodes by CIDR / ipBlock.
This requires you to configure Cilium with ``--policy-cidr-match-mode=nodes`` or
```

Reply:
@squeed I'm actually going to approve this now because I'll be OOO next week and I don't want this PR held up by my own sensibilities. Feel free to open a follow-up PR if you like.